snapcraft/plugins/python.py

```diff
@@ -159,9 +159,9 @@ def plugin_stage_packages(self):
             return
 
         stage_packages = [python_base]
-        if release_codename == 'bionic':
-            # In bionic, pip started requiring python-distutils to be
-            # installed.
+        # In bionic, python3's pip started requiring python-distutils
```

Comment:
To be correct, it should be python3-distutils

Reply:
ah, indeed.